tests/libcxx/expected_results.txt

```diff
@@ -723,6 +723,10 @@ std/containers/views/span.tuple/tuple_size.pass.cpp SKIP
 # Test bug. See LWG-3099 "is_assignable<Incomplete&, Incomplete&>"
 std/utilities/utility/pairs/pairs.pair/assign_pair.pass.cpp SKIP
 
+# Test needs to be updated for P1831R1 Deprecating volatile: library
+std/atomics/atomics.types.generic/bool.pass.cpp SKIP
+std/atomics/atomics.types.generic/integral.pass.cpp SKIP

```

Comment:
I removed the skipped_tests changes as we do not really have a rush getting this in?!

Reply:
These two tests are passing again anyway after switching to the "reference something deprecated" approach instead of the "template and SFINAE away" approach.